packages/ui/core/Button/styles.tsx

```diff
@@ -0,0 +1,109 @@
+import styled, {css} from 'styled-components/native';
+import {px2ddp} from '@animavita/theme';
+import {LinearGradient} from 'expo-linear-gradient';
+
+import Typography from '../Typography';
+
+import {ButtonActive, ButtonSize, ButtonOutline, LARGE, SMALL, ButtonRounded, ButtonDisabled} from './types';
+
+export const DisabledStyle = css``;
+
```

Comment:
I think you should have specs for this file.

Reply:
Specs like JSDoc?

Reply:
He means "specs" like tests.

Reply:
Hmm I'm not sure, because I already have tests for font and color variations in the button test.